plugins/azure/mysqlserver/enforceMySQLSSLConnection.spec.js

```diff
@@ -0,0 +1,103 @@
+var expect = require('chai').expect;
+var server = require('./enforceMySQLSSLConnection');
+
+const servers = [
+    {
+        sku: [Object],
+        location: 'eastus',
+        tags: {},
+        id: '/subscriptions/123/resourceGroups/aqua-resource-group/providers/Microsoft.DBforMySQL/servers/test-server',
+        name: 'test-server',
+        type: 'Microsoft.DBforMySQL/servers',
+        sslEnforcement: 'Enabled',
+        userVisibleState: 'Ready',
+        fullyQualifiedDomainName: 'test-server.mysql.database.azure.com',
+        earliestRestoreDate: '2021-03-14T17:53:38.68+00:00',
+        infrastructureEncryption: 'Disabled',
+        publicNetworkAccess: 'Enabled'
+    },
+    {
+        sku: [Object],
+        location: 'eastus',
+        tags: {},
+        id: '/subscriptions/123/resourceGroups/aqua-resource-group/providers/Microsoft.DBforMySQL/servers/test-server',
+        name: 'test-server',
+        type: 'Microsoft.DBforMySQL/servers',
+        sslEnforcement: 'Disabled',
+        userVisibleState: 'Ready',
```

Comment:
Use double quotes notation to define objects to keep consistent
#586

Reply:
Done as suggested.
@AkhtarAmir